### PR TITLE
Add one save button

### PR DESCRIPTION
**Pull request by szok**

### CODE REVIEW

**Comment by mkurek** [comment]
Could you record us some gif? (http://osxdaily.com/2013/08/23/record-screen-animated-gif-mac-os-x/)

**Comment by ar4s** [comment]
:ok:

**Comment by szok** [comment]
test

File: src/ralph/admin/templates/admin/submit_line.html [hunk 1]
```diff
@@ -11,36 +11,39 @@
...
+            </a>
           {% endif %}
+          {% if multi_add_field %}
+            <input type="submit" value="{% trans 'Bulk add' %}" class="default button small info" name="_multi_add" />
+          {% endif %}
+        </li>
+        <li>
+          <button type="submit" name="_continue" class="button small success split" />
+            <span data-dropdown="drop" data-options="align: top"></span>
+            {% trans 'Save and continue editing' %}
```
Comment (mkurek) [inline src/ralph/admin/templates/admin/submit_line.html]: default button should be named `Save` (keep it simple)

**Comment by mkurek** [comment]
Any new record? Does it still overflow window?

**Comment by szok** [comment]
zrzut ekranu 2015-11-26 o 14 29 28

**Comment by mkurek** [comment]
:ok:

**Pull request merged**